bank-service/src/main/java/rs/banka4/bank_service/runners/TestDataRunner.java

```diff
@@ -14,6 +15,10 @@ public class TestDataRunner implements CommandLineRunner {
 
     @Override
     public void run(String... args) {
-        dataSourceService.insertData(environment.matchesProfiles("dev"));
+        if (environment.acceptsProfiles(Profiles.of("dev"))) {
+            dataSourceService.insertData(true);
+        }
```

Comment:
this means that the database will never be seeded in production, not even with data that should be in production.  please revert.